spec/models/location_spec.rb

```diff
@@ -32,24 +32,32 @@
     end
   end
 
+  # rubocop:disable Rails/SaveBang
```

Comment:
Robocop complains that we should be using `update!` below as we're not checking the result... however we are checking it, by inspecting the errors.